ibmcloud/README.md

```diff
@@ -145,23 +173,55 @@ COS_INSTANCE_GUID=$(ibmcloud resource service-instance --output json "$IBMCLOUD_
+
+Create the `terraform.tfvars` in [the template directory](./terraform/podvm-build). `terraform.tfvars` should look like this.
 
 ```
-make push
+ibmcloud_api_key = "<your API key>"
+ibmcloud_user_id = "<IBM Cloud User ID>"
+cluster_name = "<cluster name>"
+cos_service_instance_name = "<COS Service Instance Name>"
+cos_bucket_name = "<COS Bucket Name>"
 ```
 
-After successfully uploading an image, you can verify the image by creating a virtual server instance using it. The following command will create a new server, and delete it. The VPC and subnet name are available in the terraform configuration mentioned above. You need to change the zone name if you have changed the region.
+If you used the Terraform templates in [common](./terraform/common) and [cluster](./terraform/cluster) to create the VPC and VSIs, you should set `ibmcloud_api_key` and `cluster_name` to the same values as those you entered in `terraform.tfvars` for those templates.
 
+`ibmcloud_user_id` should be set to the IBM Cloud user ID who owns the API key `ibmcloud_api_key`. You can look up the user ID using.
+
+```
+ibmcloud account users
 ```
-export IBMCLOUD_VPC_NAME=tok-vpc
-export IBMCLOUD_VPC_SUBNET_NAME=tok-primary-subnet
-export IBMCLOUD_VPC_ZONE=jp-tok-2
 
-make verify
+If `ibmcloud account users` displays multiple user IDs, choose the user ID whose state is `ACTIVE`.
+
+The `cos_service_instance_name` and `cos_bucket_name` tfvars are optional and default to values based on your `cluster_name` tfvar if not set, as per the example in the following table.
+
+| Resource | Value |
+|----------|-------|
+| `cluster_name` tfvar | peer-pod-cluster |
+| COS Service Instance name | peer-pod-cluster-cos-service-instance | 
+| COS Bucket name | peer-pod-cluster-cos-bucket |
+
+For uploading the pod VM image using Terraform, the COS Bucket must be a regional bucket in the same region (default `jp-tok`) as the VPC and VSIs.
+
+**Note:** The `Operator` and `Console Admin` roles must be [assigned](https://cloud.ibm.com/docs/vpc?topic=vpc-vsi_is_connecting_console&interface=ui) to the user. The Terraform template will create the `Console Admin` role for the user `ibmcloud_user_id` is set to in the template `terraform.tfvars`.
+
+Execute the following commands on your client machine to build, upload and verify the pod VM image.
+
+```
+cd ibmcloud/terraform/podvm-build
+terraform init
+terraform plan
+terraform apply
 ```
 
-Note that creating a server from a new image may take long time. It typically takes about 10 minutes. From the second time, creating a server from the image takes one minute.
+**Note:** if your worker node is **s390x** based, the suffix of the created QCOW2 file for the custom image will be `-s390x` otherwise it will be `-amd64`.
+
+**Note:** when verifying the pod VM image, creating a server from a new image may take long time the first time. It typically takes about 10 minutes. From the second time, creating a server from the image takes one minute.
```

Comment:
I almost cost the same time, update it to:
```
> - It typically takes about 15~20 minutes for task `Build peer pod VM image`
> - It typically takes about 7~10 minutes for task `Push peer pod VM image to Cloud Object Store and verify the image`
> - After all tasks finished, when you creating a server from the image it will only takes 1~5 minutes.
```